packages/modules/mssqlserver/src/mssqlserver-container.ts

```diff
@@ -0,0 +1,123 @@
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer } from "testcontainers";
+
+const MSSQL_PORT = 1433;
+
+type ProductKey = `${string}-${string}-${string}-${string}-${string}`;
+
+type Edition = "Evaluation" | "Developer" | "Express" | "Web" | "Standard" | "Enterprise" | ProductKey;
+
+const PASSWORD_CATEGORY_VALIDATION_PATTERNS = [/[A-Z]+/, /[a-z]+/, /[0-9]+/, /[@$!%*?&]+/];
```

Comment:
Although, this is similar to what testcontainers-java does, the current approach is to leave it to the service itself. Because, it can change and the container implementation in Testcontainers would fail when it shouldn't